lib/index.js

```diff
@@ -1,46 +1,54 @@
-var Client    = require('./client'),
-    Utils     = require('./utils'),
-    ZAFClient = {};
-
-/// ### ZAFClient API
-///
-/// When you include the ZAF SDK on your website, you get access to the `ZAFClient` object.
-///
-/// #### ZAFClient.init([callback(context)])
-///
-/// Returns a [`client`](#client-object) object.
-///
-/// ##### Arguments
-///
-///   * `callback(context)` (optional) a function called as soon as communication with
-///     the Zendesk app is established. The callback receives a context object with
-///     data related to the Zendesk app, including `currentAccount`, `currentUser`, and `location`
-///
-/// Example:
-///
-/// ```javascript
-/// var client = ZAFClient.init(function(context) {
-///   var currentUser = context.currentUser;
-///   console.log('Hi ' + currentUser.name);
-/// });
-/// ```
-ZAFClient.init = function(callback, loc) {
-  loc = loc || window.location;
-  var queryParams = Utils.queryParameters(loc.search),
-      hashParams = Utils.queryParameters(loc.hash),
-      origin = queryParams.origin || hashParams.origin,
-      app_guid = queryParams.app_guid || hashParams.app_guid,
-      client;
-
-  if (!origin || !app_guid) { return false; }
-
-  client = new Client({ origin: origin, appGuid: app_guid });
+var Client = require('./client')
+
+var Utils = require('./utils')
+
+var ZAFClient = {}
+
+/**
+ * ### ZAFClient API
+ *
+ * When you include the ZAF SDK on your website, you get access to the `ZAFClient` object.
+ *
+ * #### ZAFClient.init([callback(context)])
+ *
+ * Returns a [`client`](#client-object) object.
+ *
+ * ##### Arguments
+ *
+ *   * `callback(context)` (optional) a function called as soon as communication with
+ *     the Zendesk app is established. The callback receives a context object with
+ *     data related to the Zendesk app, including `currentAccount`, `currentUser`, and `location`
+ *
+ * Example:
+ *
+ * ```javascript
+ * var client = ZAFClient.init(function(context) {
+ *   var currentUser = context.currentUser;
+ *   console.log('Hi ' + currentUser.name);
+ * });
+ * ```
+ */
+ZAFClient.init = function (callback, loc) {
+  loc = loc || window.location
+  var queryParams = Utils.queryParameters(loc.search)
+
+  var hashParams = Utils.queryParameters(loc.hash)
+
+  var origin = queryParams.origin || hashParams.origin
+
+  var appGuid = queryParams.app_guid || hashParams.app_guid
+
+  var client
```

Comment:
Why so many line breaks?

Reply:
yeah i know, i have no idea why standard did that! This was after `standard --fix` 🤷‍♂️

Reply:
I removed these linebreaks.